### ISSUE TITLE
GIF is hard to follow

### ISSUE THREAD

**Issue by YPCrumble** [aykamko/tag#3]
I'd much rather just have plain text vs. the GIF. The input typing is too slow and the output lasts too short a time, especially the final output. Then it's a pain if you want to look at any part more than the allowed time - you have to wait for it to come around again! It would be much easier to grok if it were just plain text vs. the GIF.

**Comment by YPCrumble** [comment]
Also - other than this tiny quibble I love the idea, I appreciate you building it! :)

**Comment by aykamko** [comment]
I'm gonna keep the gif around because it's flashy, but I took your criticism and stretched out the last frame quite a bit. Hope that helps!
